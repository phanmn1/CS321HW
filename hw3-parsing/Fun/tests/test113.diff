1c1
< Lexical error at line 4, column 54.  Encountered: "." (46), after : ""
---
> Lexical error at line 4, column 20.  Encountered: "I" (73), after : ""
