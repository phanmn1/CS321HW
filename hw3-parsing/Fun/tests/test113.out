Lexical error at line 4, column 20.  Encountered: "I" (73), after : ""
